rollup.config.js

```diff
@@ -80,49 +75,79 @@ const buildPlugins = version => {
   return plugins;
 };
 
-const config = [];
+const buildConfig = (version, minify) => {
+  const plugins = buildPlugins(version, minify);
+  const minifiedExtension = minify ? ".min" : "";
 
-if (baseCode) {
-  config.push({
-    input: "src/baseCode.js",
-    output: [
-      {
-        file: `distTest/baseCode${minifiedExtension}.js`,
-        format: "cjs",
-        strict: false
-      }
-    ],
-    plugins: buildPlugins(BASE_CODE)
-  });
-}
+  if (version === BASE_CODE) {
+    return {
+      input: "src/baseCode.js",
+      output: [
+        {
+          file: `distTest/baseCode${minifiedExtension}.js`,
+          format: "cjs",
+          strict: false
+        }
+      ],
+      plugins
+    };
+  }
+  if (version === STANDALONE || version === SANDBOX) {
+    const destDirectory = version === SANDBOX ? "sandbox/public/" : "dist/";
 
-config.push({
-  input: "src/standalone.js",
-  output: [
-    {
-      file: `${destDirectory}alloy${minifiedExtension}.js`,
-      format: "iife",
-      intro:
-        "if (document.documentMode && document.documentMode < 11) {\n" +
-        "  console.warn('The Adobe Experience Cloud Web SDK does not support IE 10 and below.');\n" +
-        "  return;\n" +
-        "}\n"
-    }
-  ],
-  plugins: buildPlugins(STANDALONE)
-});
-
-if (npmPackageLocal) {
-  config.push({
-    input: "test/functional/helpers/npmPackageLocal.js",
+    return {
+      input: "src/standalone.js",
+      output: [
+        {
+          file: `${destDirectory}alloy${minifiedExtension}.js`,
+          format: "iife",
+          intro:
+            "if (document.documentMode && document.documentMode < 11) {\n" +
+            "  console.warn('The Adobe Experience Cloud Web SDK does not support IE 10 and below.');\n" +
+            "  return;\n" +
+            "}\n"
+        }
+      ],
+      plugins
+    };
+  }
+
+  // NPM_PACKAGE_LOCAL or NPM_PACKAGE_PROD
+  const filename =
+    version === NPM_PACKAGE_LOCAL ? "npmPackageLocal" : "npmPackageProd";
+
+  return {
+    input: `test/functional/helpers/${filename}.js`,
     output: [
       {
-        file: `distTest/npmPackageLocal${minifiedExtension}.js`,
+        file: `distTest/${filename}${minifiedExtension}.js`,
         format: "iife"
       }
     ],
-    plugins: buildPlugins(NPM_PACKAGE_LOCAL)
-  });
-}
+    plugins
+  };
+};
+
+const config = [];
 
+const addConfig = version => {
```

Comment:
`version` is a pretty overloaded term. Something like `variant` might be clearer.